packages-exp/auth-exp/src/platform_browser/strategies/phone.ts

```diff
@@ -73,15 +100,24 @@ export async function signInWithPhoneNumber(
   );
 }
 
+/**
+ * Links the user account with the given phone number.
+ *
+ * @param user - The user.
+ * @param phoneNumber - The user's phone number in E.164 format (e.g. +16505550101).
```

Comment:
consider making it a 415 or a 510 number :3

Reply:
lol, leaving as is